bin/anvi-dereplicate-genomes

```diff
@@ -48,56 +49,64 @@ def main(args):
 						  It cannot be less than 0 or greater than 1. Please try again")
 
 	if args.keep_ani_results and not args.ani_output_dir:
-		if args.just_do_it:
-			self.run.warning("You asked anvio to keep ANI calculation results but did not provide a location to store\
-						  them. Go ahead")
-		else:
-			raise ConfigError("You asked anvio to keep ANI calculation results but did not provide a location to store\
+		raise ConfigError("You asked anvio to keep ANI calculation results but did not provide a location to store\
 						  them :/ anvi'o is confused")
 
 
 	if args.ani_output_dir is not None and not args.keep_ani_results:
-		self.run.warning("You provided a location to store ANI results but did not ask anvi'o to store them. We\
-						 will go ahead and ignore this for now")
+		if args.just_do_it:
+			self.run.warning("You provided a location to store ANI results but did not ask anvi'o to store them. Anvi'o\
+						 will go ahead and create the path but nothing will happen")
+		else:
+			raise ConfigError("You provided a location to store ANI results but did not ask anvi'o to store them. This\
+							  doesn't make sense to us")
 
 	filesnpaths.check_output_directory(args.output_dir, ok_if_exists=False)
 	os.mkdir(args.output_dir)
-	if args.keep_ani_results and args.ani_output_dir:
+	if args.ani_output_dir:
 		filesnpaths.check_output_directory(args.ani_output_dir, ok_if_exists=False)
 		os.mkdir(args.ani_output_dir)
 
 	program = genomedistance.ANI(args)
 
 	temp_dir = program.get_fasta_sequences_dir()
 	results = program.process(temp_dir)
+	try:
```

Comment:
hey @mahmoudyousef98, it seems the indentation starts using 8 whitespaces rather than 4 in some parts of this file. I am wondering if you have changed your editor or something, but I think it would be much better to standardize this.

Reply:
Hey @meren, I'm not seeing the indentation inconsistency on my end, either on github or my remote codebase. Do you mind pointing where you see them?